pkgs/development/python-modules/torch/default.nix

```diff
@@ -356,12 +356,11 @@ in buildPythonPackage rec {
     # the following are required for tensorboard support
     pillow six future tensorboard protobuf
 
-    # ROCm build and `torch.compile` requires openai-triton
-    openai-triton
-
     # torch/csrc requires `pybind11` at runtime
     pybind11
   ]
+  # ROCm build and `torch.compile` requires openai-triton
+  ++ lib.optionals (!stdenv.isDarwin) [ openai-triton ]
```

Comment:
What do you think about https://github.com/NixOS/nixpkgs/pull/263048#issuecomment-1806066394

Reply:
I think we can do that.
I prefer ``enableTriton``, since it's possible it might be used for more than just ``torch.compile`` now or in the future.
I also think it would be best to enable it by default (or in this case, all but darwin) - There's no reason to exclude ``torch.compile`` by default.

Reply:
Ended up going with ``tritonSupport`` to match the other ``...Support`` options.